src/Workspaces/Remote/ServiceHub/Microsoft.CodeAnalysis.Remote.ServiceHub.csproj

```diff
@@ -4,7 +4,7 @@
   <PropertyGroup>
     <OutputType>Library</OutputType>
     <RootNamespace>Microsoft.CodeAnalysis.Remote</RootNamespace>
-    <TargetFrameworks>net6.0;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>net6.0-windows;netstandard2.0</TargetFrameworks>
```

Comment:
Don't we use this (or have plans to use this) on VS for Mac too?

Reply:
I assume yes, but not sure how far along those plans are and if this is currently only expected to target windows based on its current dependencies

Reply:
I wonder if it's necessary to target our remote projects to net6 instead of netcoreapp3.1 (other than the CoreComponent project, which is used to pull in dependencies for .net 6 host in VS)